<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL注入工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入带参数的URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="version_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询数据库版本</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="url_box">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dbs_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询所有数据库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="table_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>280</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询数据库所有表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="column_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询表中所有字段</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dump_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询指定字段内容</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="database_name">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>输入需要查询的数据库</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  输入需要查询的表</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="table_name">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输入需要查询的字段</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="column_name">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="result_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>201</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="url_test">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>url测试</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
